### DISCUSSION TITLE
willderness survival problems(question in chat)

### DISCUSSION THREAD

**Discussion by SecretZudan** [TownyAdvanced/Towny#5660] [Q&A] [answered]
i read the text and went in dynmap folders but there was no config.yml do i need to add it myself and paste config messages there or i need to do other thing.

**Comment by SecretZudan** [comment]
and also i want to allow block placing/breaking in wilderness, i changed and allowed that in config but still does not work. :( please also help about this. :)

**Reply by SecretZudan** [reply]
fixed it

**Reply by SecretZudan** [reply]
nvm

**Reply by SecretZudan** [reply]
you're only able to place some types. other types you aren't able to.
for example you can place dirt but not crafting table

**Comment by LlmDl** [comment, marked as answer]
How to enable wilderness use is in the [FAQ](https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions) and the install guide.

**Reply by LlmDl** [reply]
The command is /ta toggle wildernessuse
